lib/chef/audit/audit_reporter.rb

```diff
@@ -138,9 +138,13 @@ def post_auditing_data
               return
             else
               # Save the audit report to local disk
-              error_file = "failed-audit-data.json"
+              error_file = "audit-data.json"
```

Comment:
Yeah, I tend to disagree, I think the old name was fine for the people who will actually need to consume it.  I think its only because you're seeing it when you don't want it that you're getting worried about the name.  It also goes along nicely with the error message "failed to post audit..." so the meaning of failed-audit-data should be clear.